packages/fluentui/react-northstar-prototypes/src/prototypes/VirtualizedStickyTree/VirtualStickyTree.tsx

```diff
@@ -80,13 +81,54 @@ export const VirtualStickyTree: ComponentWithAs<'div', VirtualStickyTreeProps> =
     getItemById,
     registerItemRef,
     activeItemIds,
+    focusItemById: baseFocusItemById,
```

Comment:
nit
```suggestion
    focusItemById: focusItemByIdWithoutScroll,
```

Reply:
The original focusItemById actually does have scroll, but the scroll there is for classic virtual list, and doesn't take sticky into consideration.

Reply:
aaah I see, no worries leave as is